islandora_transforms/RELS-EXT_to_solr.xslt

```diff
@@ -58,14 +58,17 @@
     <xsl:template match="*[normalize-space(.)]" mode="rels_ext_element">
       <xsl:param name="prefix"/>
       <xsl:param name="suffix"/>
-
-      <xsl:call-template name="rels_ext_fields">
-        <xsl:with-param name="prefix" select="$prefix"/>
-        <xsl:with-param name="suffix" select="$suffix"/>
-        <xsl:with-param name="type">literal</xsl:with-param>
-        <xsl:with-param name="value" select="text()"/>
-      </xsl:call-template>
-    </xsl:template>
+      <!-- XXX: Parameters as passed in are strings... Let's deal with it as a
+        string here, for convenience. -->
+      <xsl:if test="string($index_compound_sequence) = 'true' or (string($index_compound_sequence) = 'false' and not(self::islandora-rels-ext:* and starts-with(local-name(), 'isSequenceNumberOf')))">
```

Comment:
`islandora-rels-ext` namespace is undefined... needs a:
```xml
    xmlns:islandora-rels-ext="http://islandora.ca/ontology/relsext#"
```
somewhere above here in the XML tree.

Reply:
[Addressed](https://github.com/discoverygarden/cuhk-basic-solr-config/pull/14/commits/66fc93f160b86812a23b8c2637804e8ac5e9ab7e#diff-9823cf446d455734ea3ac88427f23211R7).